ote/ote/tasks/mmdetection/dataset.py

```diff
@@ -0,0 +1,159 @@
+from copy import deepcopy
+import os.path as osp
+
+import numpy as np
+import mmcv
+from mmdet.datasets import DATASETS, CocoDataset
+from mmdet.datasets.pipelines import Compose
+from pycocotools.coco import COCO
+
+from ote.interfaces.dataset import IDataset
+
+
+class CocoDataSource(IDataset):
```

Comment:
In the future please create `IDetectionDataset` and describe output of getitem and get_annoation to allows 3rd party users look at specs and create a dataset that could be used with the detection task.